content/changelog/2024-07-26-domains-console-update.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Console: new Domains management interface"
+date: 2024-07-26
+tags:
+  - console
+  - domains
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Florian Sanders
+    link: https://github.com/florian-sanders-cc
+    image: https://github.com/florian-sanders-cc.png?size=40
+description: Did you know about path routing?
+excludeSearch: true
+---
+
+We've upgraded the applications' Domains page in the Console. It's now more clear, complete and easy to use. We designed it to highlight some of our underrated features such as [path routing](/doc/administrate/domain-names/#path-routing). You also now have:
```

Comment:
I suggest that instead of starting by `We've upgraded`,  we simply use `Upgrade for`. What do you think?
```suggestion
Upgrade for applications' Domains page in the Console. It's now more clear, complete, and easy to use. We designed it to highlight some of our underrated features such as [path routing](/doc/administrate/domain-names/#path-routing). You also now have:
```